jwst/regtest/test_nirspec_ifu_spec2.py

```diff
@@ -0,0 +1,45 @@
+"""Regression tests for NIRSpec"""
+import os
+
+import pytest
+
+from . import regtestdata as rt
+
+
+@pytest.fixture(scope='module')
+def run_nrs_ifu(jail, rtdata_module):
+    """Run the pipelines"""
+    step_params = {
+        'input_path': 'nirspec/spectroscopic/ifu/nrs_ifu_rate.fits',
```

Comment:
I think we can do away with the very generic "spectroscopic" subdirectory under nirspec, so that we don't then have to go down yet another directory level for each specific spectroscopic mode. I would suggest just using 'nirspec/ifu/'.

Reply:
There are other things there that will need be dealt with, but that should probably be under its own issue.